src/components/header-components/Playlist.jsx

```diff
@@ -0,0 +1,13 @@
+import PropTypes from "prop-types"
+
+export const Playlist = ({name, external_urls}) => {
+
+  return (
+      <li className="platlist-item"><a href={external_urls.spotify}>{name}</a></li>
```

Comment:
Should this maybe be playlist-item instead?
//Izabel & Linda

Reply:
Good Catch!